packaging/RedHat/rpm.spec

```diff
@@ -14,10 +14,10 @@ Summary: Client side program for distributed C/C++ compilations.
 Version: %VERSION
 Release: %rel
 Group: Development/Languages
-Url: https://code.google.com/p/distcc
+Url: https://github.com/distcc/distcc/
 License: GPL
-Packager: Google Inc. <opensource@google.com>
```

Comment:
this one is probably OK to update though, given that the distcc project will be the packager of any future rpm's (I'm not sure this is actually used... but can't hurt to update).